### PR TITLE
make access restrictions controlled

### LINKED ISSUE #959: Make access_restrictions controlled
access_restrictions is controlled according to the OD2 map, but is not set up to be controlled yet in OD2.  waiting on https://github.com/OregonDigital/oregondigital/issues/1306

### PR DESCRIPTION
**Pull request by luisgreg99**
fixes #959

### CODE REVIEW

File: app/models/concerns/oregon_digital/generic_metadata.rb [hunk 1]
```diff
@@ -719,7 +719,7 @@ module GenericMetadata
         { name: 'rights_note', is_controlled: false },
         { name: 'rights_holder', is_controlled: false },
         { name: 'copyright_claimant', is_controlled: false },
-        { name: 'access_restrictions', is_controlled: false },
+        { name: 'access_restrictions_label', is_controlled: false },
```
Comment (straleyb) [inline app/models/concerns/oregon_digital/generic_metadata.rb]: I think `is_controlled:` should be true here.

**Comment by straleyb** [comment]
Looks good, but could we get some tests for the controlled vocabulary and the endpoint? Should be some examples found here: https://github.com/OregonDigital/OD2/tree/master/spec/oregon_digital/controlled_vocabularies

File: app/models/concerns/oregon_digital/generic_metadata.rb (hunk 1 above)
Comment (luisgreg99) [reply app/models/concerns/oregon_digital/generic_metadata.rb]: done

**Comment by luisgreg99** [comment]
> Looks good, but could we get some tests for the controlled vocabulary and the endpoint? Should be some examples found here: https://github.com/OregonDigital/OD2/tree/master/spec/oregon_digital/controlled_vocabularies

Ok, I've added specs for the new vocabulary and endpoint

**Pull request merged**